### ISSUE TITLE
jdbc batch prepare statement don't use default schema set into URL

### ISSUE THREAD

**Issue by proofy** [crate/crate-jdbc#75]
if I'm setting 
 [jdbc:crate://172.16.1.64:4300/mydb](jdbc:crate://172.16.1.64:4300/mydb) , 
I would expected that 
```java
 PreparedStatement stmt = connection.prepareStatement("INSERT INTO mytable (id, createdate, info) values (?, ?, ?)");
 for (long i = 1; i < MAX_INSERTS + 1; i++) {
   stmt.setLong(1, i);
   stmt.setTimestamp(2, new Timestamp(new Date().getTime()));
   stmt.setString(3, LONG_TEXT);
   stmt.addBatch();
   if (i % MAX_FETCHSIZE == 0) {
    stmt.executeBatch();
   }
 }
```
would work with the table mydb, but I get an error message, that the table mytable is unknown.

**Comment by chaudum** [comment]
https://github.com/crate/crate-jdbc/pull/76

**Comment by chaudum** [comment]
@proofy we just released `1.8.1` which includes a fix for your issue!
https://bintray.com/crate/crate/crate-jdbc/1.8.1/view
